hazelcast/src/test/java/com/hazelcast/config/XMLConfigBuilderTest.java

```diff
@@ -87,14 +95,14 @@
             + "<action>remove</action>"
             + "</actions>";
 
-    @Test
```

Comment:
It might be better to keep `@Test` along with `@Override`. IDEA, for instance, is having troubles with running individual tests if they are overridden and not annotated with test.

Reply:
`@Test` annotations added back, thanks.